plugins/basic/nexus-repository-httpbridge/src/main/java/org/sonatype/nexus/repository/httpbridge/HttpResponseSender.java

```diff
@@ -26,5 +27,5 @@
  */
 public interface HttpResponseSender
 {
-  void send(Response response, HttpServletResponse httpServletResponse) throws ServletException, IOException;
+  void send(Response response, HttpServletRequest httpRequest, HttpServletResponse httpServletResponse) throws ServletException, IOException;
```

Comment:
Um, yes, this PR contains one unrelated change: HEAD support. And HttpResponseSender must know the HTTP method to support that. Will undo these changes and create a backlog issue.